workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -591,8 +591,8 @@ private JSONObject _transformSearchResults(JSONObject resultsJSONObject) {
 	)
 	private String _jiraSecurityVulnerabilityFieldAffectedVersionsDetails;
 
-	@Value("${liferay.customer.jira.security.vulnerability.field.category}")
-	private String _jiraSecurityVulnerabilityFieldCategory;
+	@Value("${liferay.customer.jira.security.vulnerability.field.categories}")
```

Comment:
does `application-default.properties` config file need to updated with this change as well?

Reply:
yes, I kept it "category" tho to match Jira field name (not the backend logic.)

Reply:
good catch, fixed